pkg/features/features.go

```diff
@@ -85,6 +85,11 @@ const (
 	// owner: @MichaelEischer
 	// alpha: v1.98.0
 	NewWorkerPoolHash featuregate.Feature = "NewWorkerPoolHash"
+
+	// NewVPN enables the new implementation of the VPN (go rewrite) using an IPv6 transfer network.
+	// owner: @MartinWeindel
```

Comment:
Shouldn't @ScheererJ @axel7born @DockToFuture also be added here?

Reply:
Agreed. Adding @axel7born, @DockToFuture and me makes sense.